app/containers/GuideProfilePage/index.js

```diff
@@ -11,15 +11,28 @@ import { Helmet } from 'react-helmet';
 import { createStructuredSelector } from 'reselect';
 import { compose } from 'redux';
 
-import Header from 'components/Header';
-
 import injectSaga from 'utils/injectSaga';
 import injectReducer from 'utils/injectReducer';
 import makeSelectGuideProfilePage from './selectors';
 import reducer from './reducer';
 import saga from './saga';
 
-export class GuideProfilePage extends React.Component { // eslint-disable-line react/prefer-stateless-function
+import Header from 'components/Header';
+import ProfileHeader from 'components/ProfileHeader';
+
+const sampleProfile = {
```

Comment:
The API already exists for this - you don't need to do this now but we'll need to integrate that call into the props

Reply:
Yeah, I did this right before you announced the API. I'll integrate it with the Guide Page before moving on to something else.

Reply:
I made a separate ticket for it, feel free to do that as a separate PR